iree/compiler/Dialect/VM/Conversion/VMToEmitC/test/arithmetic_ops_i64.mlir

```diff
@@ -0,0 +1,120 @@
+// RUN: iree-opt -split-input-file -pass-pipeline='vm.module(iree-convert-vm-to-emitc)' %s | IreeFileCheck %s
+
+// CHECK-LABEL: @add_i64
+vm.module @my_module {
+  vm.func @add_i64(%arg0: i64, %arg1: i64) {
+    // CHECK-NEXT: %0 = emitc.call "vm_add_i64"(%arg0, %arg1) {args = [0 : index, 1 : index]} : (i64, i64) -> i64
```

Comment:
random aside: it'd be nice to elide the args stuff if not needed - would make these calls easier to read/author.

Reply:
Those were (re-)introduced with https://github.com/google/iree/pull/4724. Yes, those don't realy help if the arg is an index but this is e.g. need to pass the `amount` arg to the shift ops. Also see @simon-camp's https://github.com/google/iree/pull/4724#discussion_r570147022.

Reply:
We can check for empty attributes on the operation during conversion and then also omit the args attribute. That's easy to do.

Reply:
As an aside to the aside, vm.shl/shr shift amount would be good to make a dynamic SSA value (vs attribute) - looking... same with vm.select's default value. Then no need for attribute handling :)

Reply:
@simon-camp As you're *supposed* to have your work finished for today let's discuss this next wee :) An dynamic SSA sounds reasonable.